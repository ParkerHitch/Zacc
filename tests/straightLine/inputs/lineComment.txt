
// Print out a funny number
print(420);

// Do some math
a0 <- 1 + 2;
a2 <- a0 * 2;

// Print result
print(a2*4);

