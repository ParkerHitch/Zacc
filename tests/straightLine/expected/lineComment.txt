PRINT: 420.0000
PRINT: 24.0000
